recipes/simplification/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+TAG="v${RDPTAG}"
+if [ ${target_platform} == "linux-64" ]; then
+	FILENAME="rdp-${TAG}-x86_64-unknown-linux-gnu.tar.gz"
+elif [ ${target_platform} == "linux-aarch64" ]; then
+	FILENAME="rdp-${TAG}-aarch64-unknown-linux-gnu.tar.gz"
+elif [ ${target_platform} == "osx-arm64" ]; then
+	FILENAME="rdp-${TAG}-aarch64-apple-darwin.tar.gz"
+elif [ ${target_platform} == "osx-64" ]; then
+	FILENAME="rdp-${TAG}-x86_64-apple-darwin.tar.gz"
+fi
+
+URL="https://github.com/urschrei/rdp/releases/download/${TAG}/${FILENAME}"
+curl -L $URL -o $FILENAME
+tar -xvf $FILENAME -C src/simplification
```

Comment:
This should be built from source and be in a separate conda package.

Reply:
`rdp` is a Rust library, can you please let me know how can we deal with this? @urschrei, who is the developer of both packages, builds `simplification` for pypi using a similar approach.

Reply:
We have several Rust packages on conda-forge, e.g. see the just merged https://github.com/conda-forge/staged-recipes/pull/25590

Reply:
Thanks for the link! I looked into it and realized there's a difference between `rdp` and the example Rust packages: Building `rdp` from source only generates one file (`librdp.dylib` for macOS, `librdp.so` for Linux, or `rdp.dll` for Windows) and stores it in the `$SRC_DIR/target/release/`. It doesn't produce any binaries.

Would it be possible, instead of creating a new `conda-forge` package, I modify the build script to generate the library file from source by downloading the source code, for example, from `https://github.com/urschrei/rdp/archive/refs/tags/v0.13.0.tar.gz`, build it with `CARGO_INCREMENTAL="0" cargo build --release`, and finally copy the generated platform specific library to `src/simplification`?